src/TorchSharp/Torch.cs

```diff
@@ -272,7 +274,7 @@ public static void InitializeDeviceType(DeviceType deviceType)
                 // If it doesn't we report the entire load trace.
                 var result = cuda.CallTorchCudaIsAvailable();
                 if (!result)
-                    throw new InvalidOperationException($"Torch device type {deviceType} did not initialise on the current machine. Trace from LoadNativeBackend:\n{trace}");
+                    throw new InvalidOperationException($"Torch device type {deviceType} did not initialise on the current machine. Trace from LoadNativeBackend:\n{trace?.ToString() ?? string.Empty}");
```

Comment:
Would it make more sense to simply not have the second sentence in the message if `trace` is null?